proxy/tests/telemetry.rs

```diff
@@ -1088,3 +1091,70 @@ mod transport {
         assert_contains!(metrics.get("/metrics"), &expected);
     }
 }
+
+// https://github.com/runconduit/conduit/issues/613
+#[test]
+#[cfg_attr(not(feature = "flaky_tests"), ignore)]
+fn metrics_compression() {
+    let _ = env_logger::try_init();
+
+    let Fixture { client, metrics, proxy: _proxy } = Fixture::inbound();
+
+    let do_scrape = |encoding: &str| {
+        let resp = metrics.request(
+            metrics.request_builder("/metrics")
+                .method("GET")
+                .header("Accept-Encoding", encoding)
+        );
+
+        {
+            // create a new scope so we can release our borrow on `resp` before
```

Comment:
Please use a complete sentence starting with a capital letter and ending with punctuation.